server/index.js

```diff
@@ -0,0 +1,37 @@
+var express = require ('express');
+var app = express();
+var bodyParser = require('body-parser');
+// var mongoose = require('mongoose');
+// // mongoose.connect();
+// var db = mongoose.connection;
+var db = require('../db')
+
+app.get('/', function(req, res) {
+  res.send('Please visit /api/item for items');
+});
+
+app.get('/api/item', function(req, res) {
+  db.getAllItems(function(err, results) {
+    if (err) {
+      console.log('There was an error: ', err);
```

Comment:
always send back a response, even if there's an error! else the client is gonna think the response is still pending